### PR TITLE
STL 2188 Botschafter Page

### PR DESCRIPTION
**Pull request by hamo225**
# Short Description
- Adds new headline and paragraph text to the Ambassador page to inform and enable relatives to assist their grandparents in using STL

# Changes
- adds new string in messages.po for the new menu footer text
- adds styling for headline text at new breakpoints
- adds new translations for headline and paragraph text
- Cleans up inline classes on the Ambassador page, adding variables to the styled-components.

# Feedback
- any?

# How to review this Pull Request
[Link to Confluence](https://digitalservice4germany.atlassian.net/wiki/spaces/STL/pages/117637157/Development+Process#Approve-or-comment) (internal)

### CODE REVIEW

File: webapp/app/translations/de/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -1102,14 +1095,6 @@ msgstr ""
 msgid "form.unlock-code-revocation.failure-title"
 msgstr "Stornierung fehlgeschlagen. Bitte prüfen Sie Ihre Angaben"
 
-#: app/forms/steps/unlock_code_revocation_steps.py:86
-msgid "form.unlock-code-revocation.failure-intro"
```
Comment (SannyNguyenHung) [inline webapp/app/translations/de/LC_MESSAGES/messages.po:1106]: this shouldn't be removed or?

File: webapp/app/translations/de/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -3148,7 +3132,7 @@ msgstr "Kontakt und Feedback"
 
 #: app/templates/basis/footer.html:22
 msgid "footer.botschafter"
-msgstr "Botschafter:in werden"
+msgstr "Über den Steuerlotsen"
```
Comment (SannyNguyenHung) [inline webapp/app/translations/de/LC_MESSAGES/messages.po:3135]: Undo this please

File: webapp/client/src/components/ContentPageStyles.js [hunk 3]
```diff
@@ -172,15 +172,20 @@ export const QuestionBoxAnchorButtons = styled.div`
 export const TopSpacing = styled.div`
   margin-top: 9.375rem;
 `;
-export const IntroHeading = styled.div`
+export const IntroHeading = styled.h1`
   font-size: 2.25rem;
-  max-width: 832px;
+  max-width: 926px;
   width: 70vw;
+  font-weight: var(--font-bold);
```
Comment (punknoir101) [inline webapp/client/src/components/ContentPageStyles.js:179]: font-family

File: webapp/client/src/components/ContentPageStyles.js [hunk 4]
```diff
@@ -241,17 +246,19 @@ export const TextContent = styled.div`
     font-size: 1.125rem;
   }
 `;
-export const ParagraphHeading = styled.div`
+export const ParagraphHeading = styled.h3`
   font-size: var(--text-2xl);
   margin-bottom: var(--spacing-03);
   max-width: 778px;
   width: 65vw;
+  font-weight: var(--font-bold);
```
Comment (punknoir101) [inline webapp/client/src/components/ContentPageStyles.js:254]: font-family

**Review by punknoir101 — CHANGES_REQUESTED** [review]

**Pull request merged**